django/conf/locale/en_IE/formats.py

```diff
@@ -0,0 +1,41 @@
+# This file is distributed under the same license as the Django package.
+#
+# The *_FORMAT strings use the Django date format syntax,
+# see https://docs.djangoproject.com/en/dev/ref/templates/builtins/#date
+DATE_FORMAT = "j M Y"  # '25 Oct 2006'
+TIME_FORMAT = "H:i"  # '14:30'
+DATETIME_FORMAT = "j M Y, H:i"  # '25 Oct 2006, 14:30'
+YEAR_MONTH_FORMAT = "F Y"  # 'October 2006'
+MONTH_DAY_FORMAT = "j F"  # '25 October'
+SHORT_DATE_FORMAT = "d/m/Y"  # '25/10/2006'
+SHORT_DATETIME_FORMAT = "d/m/Y H:i"  # '25/10/2006 14:30'
+FIRST_DAY_OF_WEEK = 0  # Sunday
```

Comment:
Are you sure it's not Monday?
- https://savvytime.com/week-number/ireland/2023
- https://www.timeanddate.com/calendar/?year=2023&country=32

Reply:
You do admittedly see calendars beginning on Monday, but in my experience calendars beginning with Sunday are by far the most common.

* https://www.irishcalendars.ie/collections/large-format-irish-calendars/products/large-real-ireland-calendar-2022